internal/monitors/kubernetes/cluster/monitor_test.go

```diff
@@ -183,6 +192,17 @@ var _ = Describe("Kubernetes plugin", func() {
 			MergeIntoExisting: true,
 		}))
 
+		Expect(dims).ShouldNot(ConsistOf(&types.Dimension{
```

Comment:
What is this test for?

Reply:
Also it is redundant to the previous assertion.

Reply:
ConsistOf will test the length as well as the elements matching.

Reply:
Oh ok. I wasn't sure if we also needed to assert that a particular Dimension should not exist in the output.

Reply:
Removed the additional check